src/Raven.Server/ServerWide/ServerMetricCacher.cs

```diff
@@ -24,7 +24,7 @@ public void Initialize()
         {
             Register(MetricCacher.Keys.Server.CpuUsage, TimeSpan.FromSeconds(1), _server.CpuUsageCalculator.Calculate);
             Register(MetricCacher.Keys.Server.MemoryInfo, TimeSpan.FromSeconds(1), CalculateMemoryInfo);
-            Register(MetricCacher.Keys.Server.MemoryInfoExtended, TimeSpan.FromSeconds(1), CalculateMemoryInfoExtended);
+            Register(MetricCacher.Keys.Server.MemoryInfoExtended, TimeSpan.FromSeconds(15), CalculateMemoryInfoExtended);
```

Comment:
We only need only when `CalculateMemoryInfo` returns that we are in low memory.
They need to be bound together.

Reply:
This is for system metrics, for notifications.

Reply:
This is used in `ServerTotalMemory`, not easy to make it rely on the MemoryInfo without more changes.

Reply:
Spoke with @grisha-kotler. He said that it should be 5 second, because this is how often we check for that value in the LowMemoryNotification thread. This was probably called more often because Studio was opened and we are requesting this value when we are in low memory.

Reply:
Let's change this to 5 seconds

Reply:
This isn't something that we operate on, however. This is used purely for reporting purposes.
I couldn't find any use of that that we actually act on.

Reply:
This is used to determine if we are in low memory (only when the non extended version returned that we are).
It's used in the low memory notification thread.

Reply:
@grisha-kotler you are incorrect I think, the low memory handler is accessing that directly, not via cacher